Automattic/AutomatticAbout-Swift
# Remove WP app logo

Remove the WordPress app logo from the Automattic Family group of apps. Mirrors https://github.com/Automattic/about-automattic-android/pull/39.

Part of https://github.com/wordpress-mobile/WordPress-iOS/pull/19477

| Before | After |
| - | - |
| <img src="https://user-images.githubusercontent.com/1898325/196306300-bd886895-1b4d-4711-8eb0-e7a3ba5541f2.png" alt="" width="350"> | <img src="https://user-images.githubusercontent.com/1898325/196306303-95984210-14a4-42de-8672-e358777a3325.png" alt="" width="350">  |

### To test

1. Run the example app
2. Verify the WP marble is no longer part of the Automattic Family section
3. Verify that other WP-related references now point to JP
4. Verify that repo documentation now uses JP instead of WP in examples
